managed/src/main/java/com/yugabyte/yw/controllers/EncryptionAtRestController.java

```diff
@@ -30,18 +33,57 @@
 import play.mvc.Result;
 
 import java.util.Base64;
+import java.util.function.Function;
 import java.util.List;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
 import java.util.Objects;
 import java.util.UUID;
 import java.util.stream.Collectors;
 
 public class EncryptionAtRestController extends AuthenticatedController {
   public static final Logger LOG = LoggerFactory.getLogger(EncryptionAtRestController.class);
 
+  private static Set<String> API_URL =
+      ImmutableSet.of("api.amer.smartkey.io", "api.eu.smartkey.io", "api.uk.smartkey.io");
+
   @Inject EncryptionAtRestManager keyManager;
 
   @Inject Commissioner commissioner;
 
+  @Inject CloudAPI.Factory cloudAPIFactory;
+
+  private void validateKMSProviderConfigFormData(ObjectNode formData, String keyProvider) {
+    if (keyProvider.toUpperCase().equals(KeyProvider.AWS.toString())
+        && (formData.get("AWS_ACCESS_KEY_ID") != null
```

Comment:
I see these constants again and again. We have them in CustomerConfigValidator.java, as example. If you/we don't like them being there, we can extract them to a proper class (in another package, maybe something like AwsConstants). But don't use simple string constants, please.

Reply:
Good point: Used consts for now. Will create AWSConstants in the follow up diff. Thanks!